### PR TITLE
Update Slider Controls example

### LINKED ISSUE #21318: Slider controls example could use value instead of textContent
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/Forms/HTML5_input_types

### What specific section or headline is this issue about?

Slider controls

### What information was incorrect, unhelpful, or incomplete?

The guide Forms > HTML5 Input Types, at the section Slider controls, give an example using the HTML tag <output> which is updated via JavaScript using: 

output.textContent = price.value;

At the end of this section exist a note pointing to a CSS-Tricks article where a similar example is given but using .value property instead of .textContent to update the DOM

Here's the link => https://css-tricks.com/the-output-element/

I'm not completely sure if the use of .textContent instead of .value will not create accessibility issues for the user. 

Quote form the output tag MDN reference:

"Many browsers implement this element as an aria-live region. Assistive technology will thereby announce the results of UI interactions posted inside it without requiring that focus is switched away from the controls that produce those results."

### What did you expect to see?

replace  output.textContent = price.value; in the example for output.value = price.value; In the case this replacement is appropiate.

### Do you have any supporting links, references, or citations?

CSS-Tricks article "The Output Element"

https://css-tricks.com/the-output-element/

MDN > References > HTML > Elements > output

Section: Accessibility Concerns

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/output#accessibility_concerns

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/forms/html5_input_types`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/Forms/HTML5_input_types
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/forms/html5_input_types/index.md
* Last commit: https://github.com/mdn/content/commit/f1f3eaf1b5a73e3d92aef40622434b5110418852
* Document last modified: 2022-09-14T13:25:46.000Z

</details>

### PR DESCRIPTION
**Pull request by dawei-wang**
Update Slider Controls example. Fixes mdn/content#21318

### CODE REVIEW

File: files/en-us/learn/forms/html5_input_types/index.md [hunk 1]
```diff
@@ -190,14 +190,14 @@ To actually display the current value, and update it as it changed, you must use
...
+output.value = price.value;
 
 price.addEventListener("input", () => {
-  output.textContent = price.value;
+  output.value = price.value;
 });
 ```
 
-Here we store references to the `range` input and the `output` in two variables. Then we immediately set the `output`'s [`textContent`](/en-US/docs/Web/API/Node/textContent) to the current `value` of the input. Finally, an event listener is set to ensure that whenever the range slider is moved, the `output`'s `textContent` is updated to the new value.
+Here we store references to the `range` input and the `output` in two variables. Then we immediately set the `output`'s `value` to the current `value` of the input. Finally, an event listener is set to ensure that whenever the range slider is moved, the `output`'s `value` is updated to the new value.
```
Comment (estelle) [inline files/en-us/learn/forms/html5_input_types/index.md:200]: while your edits are also correct, `value` is not an attribute of `<output>, and there is no mention of the HTMLOutputElement.value on the output page, or even HTMLOutputElement at all. https://developer.mozilla.org/en-US/docs/Web/API/HTMLOutputElement actually has a broken link to value. I think we may want to update other documentation to include value before we update this. 

While your edits are valid, the current is valid as well. if someone doesn't understand your suggested edits, there are no references, yet, to look up.

**Comment by sideshowbarker** [comment]
@dawei-wang Do you have any response to https://github.com/mdn/content/pull/21667#pullrequestreview-1148518383? Should we go ahead and close this?

**Comment by dawei-wang** [comment]
> 

@sideshowbarker yes let's go ahead and close this

**Comment by dawei-wang** [comment]
> @dawei-wang Do you have any response to [#21667 (review)](https://github.com/mdn/content/pull/21667#pullrequestreview-1148518383)? Should we go ahead and close this?

@sideshowbarker yes let's go ahead and close this

**Comment by sideshowbarker** [comment]
Closing per https://github.com/mdn/content/pull/21667#issuecomment-1305942089

**Comment by bsmth** [comment]
See also: https://github.com/mdn/content/pull/22285 which seems to be a duplicate

**Pull request closed without merging**